client/apps/edit/components/content2/sections/header/controls.jsx

```diff
@@ -22,15 +27,62 @@ export default class FeatureHeaderControls extends Component {
 
   getMenuColor = () => {
     const { hero } = this.props
+
     if (hero && hero.type === 'fullscreen' && hero.url && hero.url.length) {
       return 'white'
     } else {
       return 'black'
     }
   }
 
+  toggleLayoutControls = () => {
+    const { isLayoutOpen, isVideoOpen } = this.state
+
+    this.setState({
+      isLayoutOpen: !isLayoutOpen
+    })
+
+    if (isVideoOpen) {
+      this.toggleVideoControls() // hide
+    }
+
+  }
+
+  toggleVideoControls = () => {
+    const { isLayoutOpen, isVideoOpen } = this.state
+
+    this.setState({
+      isVideoOpen: !isVideoOpen
+    })
+
+    if (isLayoutOpen) {
+      this.toggleLayoutControls()
+    }
+  }
+
+  renderVideoEmbed () {
+    const { article } = this.props
+
+    if (this.state.isVideoOpen) {
+      return (
+        <div
+          className='edit-section-container'
+          data-editing
+          data-type='video'
+        >
+          <SectionVideo editing
```

Comment:
Ok just to recap: 
1. I'll skip rendering the entire video component and instead rely on `<VideoControls />`, thereby simplifying the view (since the preview video already renders at full size in the `<BasicHeader />`; no need for duplicates  
2. Move the new video toggle work outside of `<HeaderControls />` and into main index, so that it can be passed in as a child in a similar manner as the other UI items

👍